### PR TITLE
New version: Hellofont.Hellofont version 3.0.3

### PR DESCRIPTION
**Pull request by SpecterShell**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/59209)

### CODE REVIEW

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
I haven't checked the logs yet but I believe this is blocked by https://github.com/microsoft/winget-cli/issues/2057

**Comment by SpecterShell** [comment]
😕

**Comment by SpecterShell** [comment]
https://github.com/microsoft/winget-cli/issues/2057

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — DISMISSED** [review]

File: manifests/h/Hellofont/Hellofont/3.0.3/Hellofont.Hellofont.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.7-2-3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Hellofont.Hellofont
+PackageVersion: 3.0.3
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft # electron-builder
+InstallerSwitches:
+  Upgrade: --updated
```
Comment (stephengillie) [inline manifests/h/Hellofont/Hellofont/3.0.3/Hellofont.Hellofont.installer.yaml:9]: Upgrade: --updated
  Silent: /S

Comment (stephengillie) [inline manifests/h/Hellofont/Hellofont/3.0.3/Hellofont.Hellofont.installer.yaml:9]: Upgrade: --updated
Silent: /S

**Review by stephengillie — CHANGES_REQUESTED** [review]

File: manifests/h/Hellofont/Hellofont/3.0.3/Hellofont.Hellofont.installer.yaml (hunk 1 above)
Comment (SpecterShell) [reply manifests/h/Hellofont/Hellofont/3.0.3/Hellofont.Hellofont.installer.yaml:9]: The `InstallerType` has been set to nullsoft and the `/S` switch will be automatically appended by WinGet.

**Review by ItzLevvie — APPROVED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**